docs/examples/ex_f107.rst

```diff
@@ -0,0 +1,88 @@
+.. _exf107:
+
+Load F\ :sub:`10.7`\ from Multiple Sources
+==========================================
+
+As shown in :ref:`exinit`, :py:mod:`pysatSpaceWeather` has several sources that
+provide F\ :sub:`10.7`\ over different time periods.  It can be useful to
+combine these data sets into a single :py:class:`pysat.Instrument` object.
+This may be done using the
+:py:func:`~pysatSpaceWeather.instruments.methods.f107.combine_f107` function.
+
+::
+
+
+   import datetime as dt
+   import matplotlib as mpl
+   import matplotlib.pyplot as plt
+
+   import pysat
+   import pysatSpaceWeather as py_sw
+
+   f107_hist = pysat.Instrument(inst_module=py_sw.instruments.sw_f107,
+                                tag='historic', update_files=True)
+   f107_prel = pysat.Instrument(inst_module=py_sw.instruments.sw_f107,
+                                tag='prelim', update_files=True)
+   f107_fore = pysat.Instrument(inst_module=py_sw.instruments.sw_f107,
+                                tag='45day', update_files=True)
+
+   # If needed, download the data
+   f107_hist.download(start=f107_hist.lasp_stime, stop=f107_hist.today(),
+                      freq='MS')
```

Comment:
`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`

Reply:
I also get a DeprecationWarning for freq.  Same JSON error if I remove the freq kwarg.

Reply:
the following code all runs, presumably because I've previously downloaded historic f107 data.

Reply:
> I also get a DeprecationWarning for freq

Pysat gives a deprecation warning for `freq`, and pysatSpaceWeather gives a deprecation warning if you don't include `freq`.  Something to sort out.

Reply:
I created issue #86 for the freq issue.  Your error is because nothing is downloaded.  Maybe try a smaller request and see what happens?

Reply:
I deleted the 2018 data, then just asked for 2018 to today.  Still getting the same error. It did download the new data.

Reply:
> I created issue #86 for the freq issue. Your error is because nothing is downloaded. Maybe try a smaller request and see what happens?

OK, I see what's going on.  It's asking for data up to today, which results in the error.  This should be rewritten or the code should be fixed to skip days where data does not exist.

Reply:
Added a realistic end date.

Reply:
Also created Issue #87